core/transport/src/main/proto/grpc/block_worker.proto

```diff
@@ -131,6 +132,41 @@ message CacheRequest {
   optional bool async = 6;
 }
 
+// Request for load a block into alluxio
+// next available id: 4
+message LoadRequest {
+  repeated FileBlocks files = 1;
+  optional string user = 2;
+  optional int64 bandwidth = 3;
+}
+
+message FileBlocks{
+  optional string ufs_path = 1;
+  repeated int64 block_id = 2;
+  required int64 block_size = 3;
+}
+
+message LoadResponse {
+  required TaskStatus status = 1;
+  repeated BlockStatus block_status = 2;
+
+}
+
+enum TaskStatus {
+  SUCCESS = 0;
+  FAILURE = 1;
+  PARTIAL_FAILURE = 2;
+}
+
+message BlockStatus {
+  required int64 block_id = 1;
+  // The status code, which should be an enum value of [google.rpc.Code][google.rpc.Code].
+  required int32 code = 2;
+  // A developer-facing error message
+  optional string message = 3;
+  optional bool retryable = 4;
```

Comment:
let's make this required as well. Otherwise it's unclear what does missing mean.

Reply:
I'm a little concerned about back compatibility that we might want to change this field to something like retry info in the future. And default value is false so I would imagine we only specify this field when it's retryable in the very specific case